data/_ui/tooltips.txt

```diff
@@ -1410,7 +1410,7 @@ tip "Camera acceleration"
 	`Determines whether the screen center is fixed on, trails behind, or speeds ahead of the flagship while accelerating.`
 
 tip "Show CPU / GPU load"
-	`Display the CPU and GPU load at the top of the screen while in flight. Loads are presented as a percentage of a frame (1/60th of a second) that it takes to calculate or draw each frame. >100% load (or >33% when fast-forward is active) means the game will run slower.`
+	`Display the CPU and GPU load, and virtual memory usage at the top of the screen. Loads are presented as the time that it takes to calculate or draw each frame. >16.66 ms loads (or CPU load >50 ms when fast-forward is active) mean the game will run slower.`
```

Comment:
Where is this 50 ms number coming from? Fast forward calculates the game at 180 FPS. 1/180 = 5.555... ms per frame.
```suggestion
	`Display the CPU load, GPU load, and virtual memory usage at the top of the screen. CPU and GPU loads are presented as the time that it takes to calculate or draw each frame, respectively. >16.66 ms loads (or CPU load >5.55 ms when fast-forward is active) mean that the game will run slower.`
```